python/res/analysis/analysis_module.py

```diff
@@ -194,3 +194,39 @@ def initX(self, A, S, R, dObs, E, D, rng):
         X = Matrix( A.columns() , A.columns())
         self._initX(X, A, S, R, dObs, E, D, rng)
         return X
+
+    def __ne__(self, other):
+        """
+        not equal operator between two modules
+        :param other: other module to compare with
+        :return: True if different
+        """
+        return not self.__eq__(other)
+
+
+    def __eq__(self, other):
+        """
+        equality operator between two modules
+        :param other: other module to compare with
+        :return: True if the same
+        """
+
+        if self.getName()!=other.getName() or \
```

Comment:
this looks better and is more readable in analysis_config.py

Reply:
well, I think this belongs here semantically, since it compares two modules

Reply:
Corrected